test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -692,6 +699,90 @@ UCS_TEST_P(test_uct_cm_sockaddr, many_conns_on_client)
     EXPECT_EQ(num_conns_on_client, m_client_disconnect_cnt);
 }
 
+UCS_TEST_P(test_uct_cm_sockaddr, err_handle)
+{
+    UCS_TEST_MESSAGE << "Testing "     << m_listen_addr
+                     << " Interface: " << GetParam()->dev_name;
+
+    /* wrap errors since a reject is expected */
+    scoped_log_handler slh(wrap_errors_logger);
```

Comment:
we need a more specific logger wrapper which filters only reject errors, and any other error is printed and considered a failure

Reply:
but in the last test, where the client tries to connect to a bad ip address, an error is expected, that isn't reject, and it is printed but gtest shouldn't fail on it.

Reply:
so the last test should filter that specific expected error, instead of reject error

Reply:
you mean to create a new wrapper that would expect a specific rdmacm error event?

Reply:
yes, exactly

Reply:
ok, will add. in the case of a bad ip, an RDMA_CM_EVENT_ADDR_ERROR event is generated.

do we need a separate wrapper for a reject message as well?

Reply:
probably yes, since errors are different, maybe it could be possible to share their code somehow
the goal is to filter only the messages we know it's ok to get